scylla-cdc/src/log_reader.rs

```diff
@@ -0,0 +1,213 @@
+use std::sync::Arc;
+
+use anyhow;
+use chrono::Duration;
```

Comment:
Let's import the `std::time::Duration` directly and use qualified `chrono::Duration`. I think it is more common for an unqualified `Duration` to come from `std`, not `chrono`, so this should be less confusing.